### PR TITLE
Update and Split-up Help Centre + FAQ

### PR DESCRIPTION
**Pull request by paper-toaster**
## So...

**TL;DR:** Markdown sucks and I mad butthurt 

I originally began this pr with the intent of doing normal wiki cleanup on the article. But  as time went on I decided to do a bit more to help with formatting, and I ended up with this mess.

I've tried my best to reformat the Help Centre in a way that isn't an eyesore to the reader, but the limitations of Markdown just don't make that easy:
- Headers are the same color as text
- By level 5 they're already smaller than bolded text
- You can only link up to level 3 headers
- And there isn't even a clear line that separates them from the sections above

Thus, splitting up the Help Centre FAQs is the only way to make the headers linkable and the questions list-able.

---

## To-Do:

- [ ] Update links
- [ ] Reformat & organize
  - [ ] Rename articles
  - [ ] Rewrite sections that use high-level headers that cannot be linked to be linkable
  - [ ] Rewrite redirection page to list questions within every single page and their links to them
- [ ] Rewrite unnecessarily-long how-to's
- [ ] Reconsider my life choices

---

*welp, there goes my chances of getting a bounty*

### CODE REVIEW

**Comment by Ephemeralis** [comment]
please don't merge this without the review of someone from the osu!team, any changes in wording/format need to be considered especially given the design and importance of the article

will look to review it fully tomorrow

**Comment by peppy** [comment]
Updated CODEOWNERS to protect against merge fwiw

**Comment by paper-toaster** [comment]
I'll take a closer look at this article again with those concerns in mind, thank you.

**Comment by peppy** [comment]
@Ephemeralis maybe worth adding the majority of your review there into the [ASC](https://osu.ppy.sh/help/wiki/Article_Styling_Criteria) (if it's not already there)?

File: wiki/Help_Centre/Beatmapping/en.md [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+
+**You can do this through the in-game editor.**
+
+To upload your beatmap, go into the editor and hit `File` > `Upload Beatmap`.
+
+This will make your beatmap available for others to download on the website and create a dedicated thread for discussion. Updating a map is the same process for a map that has already been uploaded.
+
+Each user has a limited number of maps they can simultaneously upload. Leaving a map inactive for a month (by not updating it or not posting in its thread) will send it to the Beatmap Graveyard. To update a graveyarded beatmap, you'll need to revive it first.
+
+## The editor is telling me that the beatmap I'm trying to submit isn't mine!
```
Comment (E223) [inline wiki/Help_Centre/Beatmapping/en.md:15]: Might be a good idea to find a way to integrate [this article](https://osu.ppy.sh/help/wiki/Guides/BSS_Issues) here?

File: wiki/Help_Centre/Beatmapping/en.md [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+
+In the editor, click `File` in the menu, then `Open .osu file in Notepad`. Check the line `Creator:` and make sure that it's set to your exact username, capitalization included.
+
+If it isn't, manually change it for every difficulty in your set, reload the map, and you should be able to submit it.
+
+### I'm an osu!supporter, but I don't have 10 pending slots!
+
+**The number of slots available depends on the number of Ranked maps you have.**
+
+By default, every user has **2 pending slots**. 1 slot is added for every Ranked map the user owns, up to a total of 8 slots. Being a supporter grants 2 additional slots, from 4 (without Ranked maps), up to a total of 10.
```
Comment (E223) [inline wiki/Help_Centre/Beatmapping/en.md:27]: I think these numbers might be a bit outdated, probably a good idea to change them now. The numbers under the "Upload More Beatmaps" section of https://osu.ppy.sh/home/support should be correct.

File: wiki/Help_Centre/Beatmapping/en.md [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+
+### How can I delete my map?
+
+**The osu! team does not take requests to manually delete uploaded beatmaps.**
+
+You can delete your beatmap yourself by waiting for it to decay to the beatmap Graveyard, which it will do so after a month without updates.
+
+Once your map is in the Graveyard, you can delete it yourself by going to it's beatmap page, clicking the `Discussion` button (which will bring you to its forum page), and then clicking the `Delete` button that appears next to the beatmap link.
```
Comment (E223) [inline wiki/Help_Centre/Beatmapping/en.md:35]:
```suggestion
Once your map is in the Graveyard, you can delete it yourself by going to it's beatmap page, clicking the `Discussion` button (which will bring you to its modding discussion page), and then clicking the `Delete` button that appears next to the beatmap link.
```
I wonder if the part in parenthesis is actually needed at all? 

Also a bit unsure about the last part of the sentence. It's true that there is a Beatmap Page button right next to the delete button, but I feel like there may be a better way of describing where the delete button is.

File: wiki/Help_Centre/Gameplay/en.md [hunk 4]
```diff
@@ -0,0 +1,283 @@
...
+
+*Notice: you should make sure you are using headphones/earbuds as opposed to speakers when using the `Offset wizard` to get the best experience possible.*
+
+Every individual hardware configuration has a particular `Universal Offset` that determines whether you hear the game's audio a little later or earlier. The default values work for most people, but if you find it isn't for you, you can change it by following three simple steps:
+
+1. Go to the `Options` menu and find the `Universal offset` slider under the `Offset Adjustment` header. The default value is 0; you should only change it if needed.
+2. If the default value (0) isn't working for you, try using the `Offset wizard` button to walk you through setting a new one.
+3. If this still doesn't help, you may want to try updating your sound card or motherboard audio drivers.\
+
+## I'm experiencing lag spikes or stuttering during play!
```
Comment (E223) [inline wiki/Help_Centre/Gameplay/en.md:125]: Might make sense to link to https://osu.ppy.sh/help/wiki/Performance_Troubleshooting somewhere in this section.

File: wiki/Help_Centre/Gameplay/en.md [hunk 5]
```diff
@@ -0,0 +1,283 @@
...
+
+**Connecting too many devices to one port via a splitter can cause input lag, which is hard to detect, but can seriously throw off your game.**
+
+An easy solution to this is to try unplugging your USB hub briefly. If your lag disappears, you'll know why!
+
+### Are you running osu! in Windowed or Borderless mode?
+
+**Both Windowed and Borderless mode introduce a *considerable* amount of input delay.** 
+
+We reccomend using Fullscreen as it is optimal for the smoothest gameplay experience. <!-- this line could be bolded with the line above, or stay down here, I can't decide tho (remove comment before merging) -->
```
Comment (E223) [inline wiki/Help_Centre/Gameplay/en.md:174]:
```suggestion
We recommend using Fullscreen as it is optimal for the smoothest gameplay experience. <!-- this line could be bolded with the line above, or stay down here, I can't decide tho (remove comment before merging) -->
```

File: wiki/Help_Centre/Gameplay/en.md [hunk 6]
```diff
@@ -0,0 +1,283 @@
...
+
+If you are using windowed or borderless mode to sync up your monitor's refresh rate, we reccomend you try selecting the `VSync` option under `Frame Limiter` first! <!-- this line and the line below are only one sentence long but they don't really fit together on the same line (remove comment before merging) -->
```
Comment (E223) [inline wiki/Help_Centre/Gameplay/en.md:176]:
```suggestion
If you are using windowed or borderless mode to sync up your monitor's refresh rate, we recommend you try selecting the `VSync` option under `Frame Limiter` first! <!-- this line and the line below are only one sentence long but they don't really fit together on the same line (remove comment before merging) -->
```

File: wiki/Help_Centre/Gameplay/en.md [hunk 7]
```diff
@@ -0,0 +1,283 @@
...
+
+If you are using windowed mode to lower the "sensitivity" of your tablet, we reccomend that you try adjusting your tablet area (which you can learn to do in the "Performance" section of this article!).
```
Comment (E223) [inline wiki/Help_Centre/Gameplay/en.md:178]:
```suggestion
If you are using windowed mode to lower the "sensitivity" of your tablet, we recommend that you try adjusting your tablet area (which you can learn to do in the "Performance" section of this article!).
```
Also, the performance section is no longer on the same page as this one, so it should probably be linked to directly instead of just mentioned. Not sure if we want to change the wording of the bit in parenthesis at the same time, so leaving that out of the suggested change.

File: wiki/Help_Centre/Performance/en.md [hunk 8]
```diff
@@ -0,0 +1,110 @@
...
+
+All osu! branded tablets support raw input by default, and may have their sensitivity easily adjusted on the fly by using using the mouse sensitivity slider. Unfortunately,<!--as of ____,--> osu! branded tablets are no longer being officially sold or manufactured.
+
+### My songs list keeps scrolling!
+
+**This is usually due to a faulty input device connected to your computer. Try unplugging any controllers or joysticks.**
+
+Applications which emulate or change keybindings such as *Xpadder* or *X-Mouse Button Control* may also cause these issues. Try disabling them if you use them for other games.
+
+This can also happen due to issues with the numpad keys, as they can be used to scroll the song selection list. Pressing the `NumLock` key to disable the numpad and then pressing either `9` or `3` on the numpad will fix the issue. You should also check for any jammed or damaged buttons on your peripherals while doing this.
```
Comment (E223) [inline wiki/Help_Centre/Performance/en.md:102]: 8 and 2 on the numpad can also fix this issue, not just 9 and 3 (can be determined based on how fast the list is scrolling). Pressing the up/down arrow keys themselves can also fix this in the case where 8 and 2 would have worked. Might be a good idea to include either one of the two? Not entirely sure how to word it though.

File: wiki/Help_Centre/Beatmapping/en.md (hunk 1 above)
Comment (paper-toaster) [reply wiki/Help_Centre/Beatmapping/en.md:15]: Linking it with a hatnote would prob be more helpful

File: wiki/Help_Centre/Beatmapping/en.md (hunk 3 above)
Comment (paper-toaster) [reply wiki/Help_Centre/Beatmapping/en.md:35]: I think just knowing that the delete button is on that page and not hidden behind any dropdown menus is enough for most people. If it's really that vague we can add an image or something.

File: wiki/Help_Centre/Performance/en.md (hunk 8 above)
Comment (paper-toaster) [reply wiki/Help_Centre/Performance/en.md:102]: I think a note including that information would would work fine here
